test/unit/specs/utils/license.spec.ts

```diff
@@ -39,3 +39,24 @@ describe('getFullLicenseName', () => {
     }
   )
 })
+
+describe('getElements', () => {
+  it.each`
+    license           | icons
+    ${'by'}           | ${['cc', 'by']}
+    ${'by-sa'}        | ${['cc', 'by', 'sa']}
+    ${'by-nd'}        | ${['cc', 'by', 'nd']}
+    ${'by-nc'}        | ${['cc', 'by', 'nc']}
+    ${'by-nc-sa'}     | ${['cc', 'by', 'nc', 'sa']}
+    ${'by-nc-nd'}     | ${['cc', 'by', 'nc', 'nd']}
+    ${'nc-sampling+'} | ${['cc', 'nc', 'sampling-plus']}
+    ${'sampling+'}    | ${['cc', 'sampling-plus']}
+    ${'pdm'}          | ${['pd']}
+    ${'cc0'}          | ${['cc', 'zero']}
```

Comment:
If this is such a 1:1 mapping I wonder if we shouldn't just create a `Record<License, LicensePart[]>` object. We're basically already hand-writing that mapping for the tests anyway (which we could safely remove if it was just a frozen configuration object :wink: )

Up to you though!

Reply:
That's a valid point, but for now I want to keep the code lean and tests verbose. Can revisit this in the future though, especially if we add licenses that don't follow this pattern.

Reply:
I'd probably describe an object mapping one to the other more lean than the current code which has a few branches of conditional logic.

But totally up to you if you prefer the imperative approach to a declarative one for this :+1: